### PR TITLE
docs(sdk): Update package_readme.md for PyPi

### PR DESCRIPTION
**Pull request by ngrayluna**
Description
-----------
Ticket: https://wandb.atlassian.net/browse/DOCS-936

What does the PR do? Include a concise description of the PR contents.

- [X] I updated CHANGELOG.md, or it's not applicable

Testing
-------
How was this PR tested?

### CODE REVIEW

File: package_readme.md [hunk 1]
```diff
@@ -4,92 +4,83 @@
...
 
-def training_loop():
-    while True:
-        # Do some machine learning
-        epoch, loss, val_loss = ...
-        # Framework agnostic / custom metrics
-        wandb.log({"epoch": epoch, "loss": loss, "val_loss": val_loss})
+# Log metrics over time to visualize performance with wandb.log
+for i in range(10):
+    run.log({"loss": loss})
```
Comment (kptkin) [inline package_readme.md:49]: could you add `run.finish()`? thanks

File: package_readme.md [hunk 2]
```diff
@@ -4,92 +4,83 @@
...
+Be a part of the growing W&B Community and interact with the W&B team in our [Discord](https://wandb.me/discord). Stay connected with the latest ML updates and tutorials with [W&B Fully Connected](https://wandb.ai/fully-connected).
 
-## Community
-Got questions, feedback or want to join a community of ML engineers working on exciting projects?
+&nbsp;
 
-<a href="https://bit.ly/wb-slack"><img src="https://svgshare.com/i/M93.svg" alt="slack" width="55"/></a> Join our [slack](https://bit.ly/wb-slack) community.
+# License
 
-[![Twitter](https://img.shields.io/twitter/follow/weights_biases?style=social)](https://twitter.com/weights_biases) Follow us on [Twitter](https://twitter.com/weights_biases).
```
Comment (kptkin) [inline package_readme.md:95]: oh we don't want to link our twitter anymore?

Comment (dmitryduev) [reply package_readme.md:95]: x?

File: package_readme.md (hunk 1 above)
Comment (dmitryduev) [inline package_readme.md:49]:
```suggestion
    run.log({"loss": loss})

# Mark a run as finished, and finish uploading all data    
run.finish()
```

**Review by dmitryduev — APPROVED** [review]
❤️

**Pull request merged**